typings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ytdl-core",
+  "main": "./lib/index.d.ts",
+  "dependencies": {},
+  "globalDependencies": {
+    "node": "registry:dt/node#6.0.0+20160613154055"
```

Comment:
You have to remember that `typings` and the `typings.json` is only used when a developer is **contributing** to `node-ytdl-core`, not when it is used within another node app (or when unit tests are ran on Travis CI). The version _could_ be rolled back to `4.0.0`, but that really doesn't do much, since the utilized API for `http.ClientRequest` and `stream.Readable` hasn't changed between `4.0.0` and `6.0.0`.

It is very difficult for me to explain how to develop tools to make it easier to develop tools, but [maybe this explanation will help](https://github.com/typings/typings/blob/master/docs/faq.md#why). Maybe the best idea is to just pull it down, and mess around (delete some files, change some code, removed dependencies, etc). That will really give you a good idea as to what each files does.